ci/ray_ci/automation/determine_microcheck_tests.py

```diff
@@ -6,14 +6,14 @@
 from ray_release.test import Test
 from ray_release.result import ResultStatus
 
-LINUX_PYTHON_TEST_PREFIX = "linux:__python"
+LINUX_TEST_PREFIX = "linux:__"
```

Comment:
this now feels like a quite weird prefix string.. maybe add a comment on why this looks like this.